client/src/components/PopoverOptionSet/PopoverOptionSet.scss

```diff
@@ -0,0 +1,94 @@
+.popover-option-set {
+  min-width: 320px;
+  display: flex;
+  flex-direction: column;
+  border-radius: $border-radius; // Change to equal 4px
+
+  &:focus,
+  &:hover {
+    box-shadow: $z-depth-1;
+    outline-width: 0;
+  }
+
+  &__button-container {
+    width: 100%;
+    padding: 10px;
+    box-shadow: $z-depth-1;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    max-height: 305px;
+    overflow-y: auto;
+
+    :nth-child(odd) {
+      margin-right: $panel-padding-x/2;
+    }
+
+    :nth-child(even) {
+      margin-right: 0;
+    }
+
+    :nth-last-child(-n+2) {
+      margin-bottom: 0;
+    }
+
+    .popover-option-set__no-results {
+      text-align: center;
+      vertical-align: middle;
+      width: 100%;
+      margin: 0;
+      padding: $panel-padding-x/2;
+    }
+  }
+
+  &__button {
+    width: calc(100% * (1 / 2) - 5px);
+    padding: $panel-padding-x/2;
+    text-align: start;
+    margin-bottom: 4px;
+    border: 0;
+    max-height: 40px;
+  }
+
+  // Adds necessary specificity to override default font-size and margins
+  & &__button::before {
+    font-size: 2.1rem;
+    margin-right: $panel-padding-x/2;
+  }
+
+  &__search {
+    height: $toolbar-height;
+
+    &-input,
+    &-clear {
+      &,
+      &:focus,
+      &:hover,
+      &:active {
+        box-shadow: none;
+        border: 0;
+        border-bottom: 1px solid $border-color-light;
+        border-radius: 0;
+        outline: none;
+      }
+    }
+
+    &-input {
+      padding-left: $panel-padding-x;
+    }
+
+    // Nesting needed for specificity, adds padding to clear link
+    .input-group-append {
+
```

Comment:
I definitely feel like this should be a lint error =/ I've seen issues where our SCSS linting it just not working at all before, could you make test to make sure this is valid?